matrix_sdk/src/client.rs

```diff
@@ -447,7 +447,7 @@ impl Client {
     ///
     /// # Arguments
     ///
-    /// * `session` - An session that the user already has from a
+    /// * `session` - A session that the user already has from a
```

Comment:
Can't this be a separate commit? Many Git UIs mangle commit messages if they are too long, and you seem to have 3 commits going on here.